Maple2.File.Tests/AiParserTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Maple2.File.Parser.Xml.AI;
+using Maple2.File.Parser;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using static System.Runtime.InteropServices.JavaScript.JSType;
```

Comment:
weird import

Reply:
The JSType one? Idk whats up with it either lol

Reply:
Yea just remove it?